### PR TITLE
ci: automatically update github actions via dependabot

### PR DESCRIPTION
**Pull request by xtreem88**
## Changes:

- Added Dependabot configuration to monitor GitHub Actions.
- Set to check updates daily and auto-create PRs for any found updates.

## Benefits:

- Keeps our actions updated and secure.
- Automates maintenance, reducing manual effort.

### CODE REVIEW

File: .github/workflows/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: 2
```
Comment (levibostian) [inline .github/workflows/dependabot.yml:1]:
```suggestion
# Dependabot helps update dependencies to keep them up-to-date. 
# Configuration docs: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
version: 2
```

Would be nice to have a link to the config file docs.

File: .github/workflows/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "hourly"
```
Comment (levibostian) [inline .github/workflows/dependabot.yml:6]:
```suggestion
      interval: "weekly"
```

I don't think [hourly is an option](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval). 

How about weekly? In the past, I have found daily to be a bit overwhelming where dependabot makes a lot of PRs and it's annoying to keep up with.

File: .github/workflows/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    target-branch: "cleaning/dependabot"
```
Comment (levibostian) [inline .github/workflows/dependabot.yml:7]:
```suggestion
```

`target-branch` is the branch where code exists that should be updated. For this repo, that's `beta`. Dependabot uses the repo's default branch as the default value so we shouldn't need to specify anything.

File: .github/workflows/dependabot.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    commit-message:
+      prefix: "chore:"
```
Comment (levibostian) [inline .github/workflows/dependabot.yml:9]:
```suggestion
      prefix: "chore"
```

The docs make it sound like the colon is added for us. No need to specify it.

**Review by levibostian — COMMENTED** [review]
:excited:!

File: .github/workflows/dependabot.yml (hunk 2 above)
Comment (levibostian) [inline .github/workflows/dependabot.yml:6]:
```suggestion
    schedule:
      interval: "weekly"
    reviewers:
      - "customerio/mobile"
```

How about automatically adding mobile group as reviewers?

Comment (xtreem88) [reply .github/workflows/dependabot.yml:6]: Yes, it is going to be weekly, I set it to hourly so that we can test it

**Review by levibostian — APPROVED** [review]

**Pull request merged**